examples/imagenet/train.py

```diff
@@ -73,8 +76,9 @@ def init(*args):
 
 
 def cross_entropy_loss(logits, labels):
-  return -jnp.sum(
-      common_utils.onehot(labels, num_classes=1000) * logits) / labels.size
+  xentropy = optax.softmax_cross_entropy(
```

Comment:
In `imagenet/models.py` we return `nn.log_softmax(x)`:

https://github.com/google/flax/blob/d804b906b803d8a30ec7f60bec3e66b247adbe15/examples/imagenet/models.py#L117

In terms of numerics, do you know if it makes any difference computing that twice?

Should we remove it from `models.py` in any case?

Reply:
Removed nn.log_softmax(x)